lib/tennis_set.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'tennis_game'
+require_relative 'tennis_score'
+
+# defines a tennis set, and handles all scoring for the game
+class TennisSet < TennisScore
+  def initialize(skip_games, players)
+    super(6, 2, 'set', players)
```

Comment:
The 2 next to it means they must win by a margin of 2, but there's no limit to the number of games they can play, (as long as the margin is never more than one) which I'm guessing is wrong?